src/python/pants/core/util_rules/environments.py

```diff
@@ -175,6 +178,56 @@ class LocalEnvironmentTarget(Target):
     )
 
 
+class LocalWorkspaceFallbackEnvironmentField(FallbackEnvironmentField):
+    help = help_text(
+        f"""
+        The environment to fallback to when this local workspace environment cannot be used because the
```

Comment:
what's an example use case for this fallback? I'm wondering if it's strictly necessary?

Reply:
Hmm, yeah maybe we should not support fallback? Workspace execution is too different from local, remote, and docker execution to be considered similar.

Reply:
I removed the fallback field from `workspace_environment`. So no fallback for workspace environments.